tsconfig.eslint.json

```
@@ -21,6 +21,7 @@
       "@bitwarden/billing": ["./libs/billing/src"],
       "@bitwarden/common/*": ["./libs/common/src/*"],
       "@bitwarden/components": ["./libs/components/src"],
+      "@bitwarden/generator": ["./libs/tools/generator/core/src"],
```

⛏️ `generator-core` is what's in the package.json

```suggestion
      "@bitwarden/generator-core": ["./libs/tools/generator/core/src"],
```